libraries/core-react/src/Popover/Popover.tsx

```diff
@@ -15,35 +13,59 @@ const Anchor = styled.div`
     outline: none;
   }
 `
+type PopoverChild = {
+  type?: { displayName?: string }
+} & ReactElement
+
+export type Props = {
+  /* Popover placement relative to anchor */
+  placement?:
+    | 'topLeft'
+    | 'top'
+    | 'topRight'
+    | 'rightTop'
+    | 'right'
+    | 'rightBottom'
+    | 'bottomLeft'
+    | 'bottom'
+    | 'bottomRight'
+    | 'leftTop'
+    | 'left'
+    | 'leftBottom'
+  /**  On Close function */
+  onClose?: () => void
+  /**  Open activates <PopoverItem/> */
+  open?: boolean
+  children?: PopoverChild | PopoverChild[]
```

Comment:
Missing description about children. 

Specifying PopopverChild (with ReactElement) seems very strict compared to our other components. Fragments, nullish, text and such are not approved children with this type.

Reply:
This was actually a workaround to make the if / else if in the component work 🙈  Would it be better if we use ReactNode or do you have a completely different suggestion @mimarz ?

Reply:
@wenche Figured as much, checkout my comment [bellow](https://github.com/equinor/design-system/pull/695#discussion_r509075348) to see if that helps for the workaround while keeping childre `ReactNode`